tensorflow_serving/model_servers/main.cc

```diff
@@ -265,6 +272,42 @@ class PredictionServiceImpl final : public PredictionService::Service {
   bool use_saved_model_;
 };
 
+class ModelManagement final : public ModelManagementService::Service {
+  public:
+    explicit ModelManagement(ServerCore *core)
+            : core_(core) {}
+
+    grpc::Status Load(ServerContext *context,
+                      const LoadModelConfigurationRequest *request,
+                      LoadModelConfigurationResponse *response) override {
+      const tensorflow::serving::ModelConfigList &list = request->configuration().model_config_list();
+      LOG(INFO) << "Reloading server config";
+
+      for (int index = 0; index < list.config_size(); index++) {
```

Comment:
Let's drop this logging. Or move it inside ReloadConfig() (perhaps in a separate PR)